receiver/riakreceiver/integration_test.go

```diff
@@ -0,0 +1,132 @@
+// Copyright  The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
```

Comment:
You'll need to add build tags here to ensure this only runs for integration tests. ([example](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/receiver/apachereceiver/integration_test.go#L15-L16))

Also, add the module to [this list](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/Makefile#L19-L31) so that it runs the test.